react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,19 +1,25 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick = {() => this.props.onClick(i)}
+      />
+    );
   }
 
+
   render() {
-    const status = 'Next player: X';
     return (
-      <div>
-        <div className={styles.status}>{status}</div>
+      <React.Fragment>
```

Comment:
You can write:

 import React, { Component, Fragment } from 'react';

And then you don't need to write React.Fragment